### PR TITLE
Aliases configurable

### PR DESCRIPTION
**Pull request by Maxthegreat99**
What i made needs like, a huge rework, and uh some docs but see if it makes sense still

ok so i

- sent Helper.cs to the shadow realm and gave housing to its methods in Api.cs
- reworked BankAccounts, now we serialize a dictionary into the database that represent the Currency/Balance KPV, uh need to see how its working
- now we can log actual strings in our logs, implemented the method with creating and purging accounts
- we are now caching our bank accounts (CONCURENCY AAAAAAAA)
- old trans log get sent to brazil if theres too many files, can be configured
- Account's money get sent to brazil if they change world, can be configed, theres an ignore perm, doesnt affect world acc
- System currency has been implemented, still need to implement currency uh transaction fluctuation thing

### CODE REVIEW

**Comment by Maxthegreat99** [comment]
ok so uhm

**Comment by Maxthegreat99** [comment]
dont merge it i need to accomodate to your PR

File: TSEconomy/Api/Api.cs [hunk 1]
```diff
@@ -1,17 +1,25 @@
-﻿using PetaPoco;
+﻿using IL.Terraria.Graphics;
```
Comment (RenderBr) [inline TSEconomy/Api/Api.cs]: whats using this

**Comment by Maxthegreat99** [comment]
its good for review i think

**Review by RenderBr — APPROVED** [review]

**Pull request merged**